ui/app/models/oidc/provider.js

```diff
@@ -2,7 +2,7 @@ import Model, { attr, hasMany } from '@ember-data/model';
 import lazyCapabilities, { apiPath } from 'vault/macros/lazy-capabilities';
 
 export default class OidcProviderModel extends Model {
-  @hasMany('identity/oidc/scope') scopes_supported;
+  @hasMany('oidc/scope') scopes_supported;
```

Comment:
I'm a bit confused by that since I've always aligned the property keys with what is returned from the server. I see on line 7 of the application adapter it's decamelizing the key:
```js
keyForAttribute: function (attr) {
  return decamelize(attr);
}
```
Is this something Ember Data just does? I didn't catch on to that and the recent models I've created I've used snake_case without issue. I agree though that we should keep it consistent. It just seems strange that the keys come in as snake_case and we need to convert them back before making the request. Seems like an unnecessary step.

Reply:
Hm - this might be a good thing to bring up to the rest of the team. It was something shared/taught to me when I first started. So I just did it without questioning 😅